docs/file-formats/nirvana-json-file-format.mdx

```diff
@@ -30,38 +32,38 @@ For each gene that was referenced in the transcripts found in the positions sect
 
 ## Header
 ```json
-{ 
-   "header":{ 
+{
```

Comment:
I'm a bit confused about all the changes in this file... seems like a ton of lines were changed... Never mind, this was just a line ending change. No worries...